allauth/utils.py

```diff
@@ -316,3 +318,17 @@ def get_request_param(request, param, default=None):
     if request is None:
         return default
     return request.POST.get(param) or request.GET.get(param, default)
+
+
+def generate_code_challenge():
+    # minimum length of 43 characters, maximum length of 128 characters
+    # each nbytes is 1.3 characters; 128/1.3 = 98
```

Comment:
>     # each nbytes is 1.3 characters; 128/1.3 = 98
Not sure I understand this comment?

Reply:
I originally had the ability to provide a length into this function and then calculated what `nbytes` had to be based on the length and 1.3 characters per byte.  Then I decided it wasn't necessary to allow passing a length in and removed that but didn't remove the comment in case it was useful in the future.